public/lib/column-defaults.js

```diff
@@ -276,6 +297,14 @@ var columnDefaults = [{
     template: lastModifiedByTemplate,
     active: false,
     isNew: true
-}].map(col => col.labelHTML === '' ? {...col, labelHTML: '&nbsp;'} : col);
+}].map(col => {
+  if (col.labelHTML === '') {
+    return {...col, labelHTML: '&nbsp;'};
+  }
+  if (col.isSortable) {
```

Comment:
Does the `return` above mean you need a `labelHTML` to be eligible for sorting?

Reply:
It does! Coincidentally, that's true of the columns we currently sort by, but it's not what we want, thanks for spotting 👍

Reply:
(hmm, on further inspection would we ever want to sort by headings that aren't visible? IDK, but I'll add it as an option anyway)

Reply:
> would we ever want to sort by headings that aren't visible

Interesting point! What happens if the active sort column is removed? I would say: this should be the only instance of thi _none_ (default sort state): sorting should revert (and be indicated against) default sort column. In case default sort column is itself removed… hmm… shiiiite. (next one right? haha)